chart/values.yaml

```diff
@@ -284,6 +288,9 @@ io_engine:
   nodeSelector:
     openebs.io/engine: mayastor
     kubernetes.io/arch: amd64
+  # -- Pod scheduling priority
+  # ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+  priorityClassName: ""
```

Comment:
IMO, this is not needed, make use of the global field and refer the same in the io-engine-daemonset yaml template.

Reply:
Updated.